source/integrate/servicenow-interoperability.rst

```diff
@@ -0,0 +1,150 @@
+Connect ServiceNow to Mattermost
+=================================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. |product-menu| image:: ../images/products_E82F.svg
+  :alt: The Product menu is located in the top left corner of the Mattermost screen.
+  :class: theme-icon
+
+Minimize distractions and reduce context switching by bridging the gap between IT service management (ITSM) and team communication. Create and manage incident reports, change requests, and service tickets, as well as manage event-driven notification subscriptions for ServiceNow record changes, in real-time, and automate routine tasks to decrease response times without leaving Mattermost.
+
+Setup
+------
+
+Setup starts in ServiceNow and finishes in Mattermost.
+
+Create an OAuth app in ServiceNow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to your ServiceNow instance and then to **All > System OAuth > Application Registry**.
+2. Select New in the top right corner, and then select **Create an OAuth API endpoint for external clients**.
+3. Enter the name for your app and set the redirect URL to: ``https://<YOUR-MATTERMOST-URL>/plugins/mattermost-plugin-servicenow/api/v1/oauth2/complete``, replacing ``YOUR-MATTERMOST-URL`` with the Mattermost URL you want the ServiceNow events to post to, using lowercase characters.
+
+ .. note:: 
+
+  A client secret is generated automatically. Copy the secret and the Client ID. You'll need these values for the Mattermost configuration.
+
+Upload the update set in ServiceNow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changing your ServiceNow instance to support subscriptions, record changes, and send change notifications to Mattermost is done by a Mattermost and ServiceNow system admin using an `update set <https://docs.servicenow.com/bundle/sandiego-application-development/page/build/system-update-sets/concept/system-update-sets.html>`_. Once created, you can download the update set from Mattermost and upload it into ServiceNow. 
```

Comment:
I think this docs link may have broken on SN's side

Reply:
Thanks for this heads-up! I'll investigate and fix.